### PR TITLE
open-cluster-management/backlog#12069 Add 217 errata

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -5,6 +5,18 @@ By default, Errata updates are automatically applied when released. See link:../
 
 *Important:* For reference, link:https://access.redhat.com/errata/#/[Errata] links and GitHub numbers might be added to the content and used internally. Links that require access might not be available for the user.
 
+== Errata 2.1.7
+
+View a summarized list of {product-title} Errata 2.1.7 updates:
+
+* *Important:* You must be runnning {ocp} version 4.6, or later, to upgrade to {product-title-short} version 2.1.7. If you cannot upgrade your {ocp} version 4.5 to a later version, you can continue to use {product-title-short} version 2.1.6.
```
Comment (swopebe) [inline release_notes/errata.adoc]: Please change to: You must run (vs be running)

**Review by blkingrh — APPROVED** [review]

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -5,6 +5,18 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixed an issue that caused the Hive controller log to show the incorrect version information. (GitHub 12014)
```
Comment (dockerymick) [inline release_notes/errata.adoc:14]: Change Fixed to Fixes (or maybe not, I believe we agreed to use present tense for 2.2 onward

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -5,6 +5,18 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Added authorization for users with `View` permissions to create and delete the `ManagedClusterView` resource, which also enables users with `View` permissions to access to view the YAML file of the managed cluster resources. (GitHub 11243)
```
Comment (dockerymick) [inline release_notes/errata.adoc]: `view` permissions

also remove "to access"

**Review by dockerymick — APPROVED** [review]
I added a reference link to the RBAC topic. I made a couple of suggestions as well. Other than that, this looks good to me. Thanks Chris!

File: release_notes/errata.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/errata.adoc:14]: Yeah, he wanted to avoid going back and I support that! LOL

**Review by swopebe — CHANGES_REQUESTED** [review]
small change requested

**Comment by cadawson** [comment]
Thank you for the reviews!

**Pull request merged**